build.yaml

```diff
@@ -397,17 +409,19 @@ common_cli:
     # which are made available through a webserver running on port 80.
     # We pass the URL of our Jenkins Slave to the acceptance test nodes
     # through the --build-server parameter below.
-    parse_logs \$venv/bin/python admin/run-acceptance-tests \
+    parse_logs \${venv}/bin/python admin/run-acceptance-tests \
     --distribution \${DISTRIBUTION_NAME} \
     --provider aws --dataset-backend aws --branch \${TRIGGERED_BRANCH} \
     --build-server  \
     http://\$(wget -qO- http://instance-data/latest/meta-data/public-ipv4)  \
     --config-file /tmp/acceptance.yaml \
-    \${ACCEPTANCE_TEST_MODULE} ; RC=\$?
+    \${ACCEPTANCE_TEST_MODULE}
+    JOB_EXIT_STATUS="\$( updateExitStatus \$? )"
 
   run_acceptance_rackspace_tests: &run_acceptance_rackspace_tests |
     # We gather the return code but make sure we come out of these tests with 0
-    # we store that code into RC and pass it to the end of the job execution.
+    # we store that code and pass it to the end of the job execution,
+    # as part of the JOB_EXIT_STATUS variable.
     #
```

Comment:
make sure JOB_EXIT_STATUS will contain 0 if every thing is working fine until now